src/client/common/process/types.ts

```diff
@@ -25,6 +25,7 @@ export type SpawnOptions = ChildProcessSpawnOptions & {
     throwOnStdErr?: boolean;
     extraVariables?: NodeJS.ProcessEnv;
     outputChannel?: OutputChannel;
+    pytestExecutionTestIds?: string[];
```

Comment:
```suggestion
    stdinStr?: string;
```

That will make it a generic way of passing string content to a process. You could even make that a stream, but that us not necessary.